docs/fastddsgen/pubsub_app/includes/idl_file.rst

```diff
@@ -1,12 +1,11 @@
-First of all you need to follow the steps outlined in the Installation Manual for the installation of
-*eprosima Fast RTPS* and all its dependencies.
-If you have followed the steps, you should have Fast DDS, Fast CDR, and Fast-RTPS-Gen installed.
+Creating the IDL file with the data type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To build a minimal application, you must first define the Topic.
 To do this, the data type of the Topic is defined by means of an IDL file.
 An example of this is shown below. In this case, a single string is sufficient.
 Topics are explained in more detail in :ref:`dds_layer_topic`, while the topic data types to be defined using IDL are
-presented in :ref:`fastrtpsgen_intro`.
+presented in :ref:`dds_layer_definition_data_types`.
```

Comment:
This paragraph is a little strange to read, a lot of very short sentences that may be combined in longer ones